google-cloud-recaptcha_enterprise/samples/create_assessment.rb

```diff
@@ -0,0 +1,70 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START recaptcha_enterprise_create_assessment]
+
+require "google/cloud/recaptcha_enterprise/v1"
+
+def create_assessment
+  ### Create an assessment to analyze the risk of a UI action.
+
+  # Args:
+  # project_id: GCloud Project ID
+  # site_key: Site key obtained by registering a domain/app to use recaptcha services.
+  # token: The token obtained from the client on passing the recaptcha site_key.
+  # recaptcha_action: Action name corresponding to the token.
+  # assessment_name: Specify a name for this assessment.
+  ###
+
+  site_key = "your_site_key"
+  token = "user_response_token"
+  project_id = "your_project_id"
+  recaptcha_action = "action_name"
+  assessment_name = "assessment_name"
+  project_name = "projects/#{project_id}"
+
+  # Create the reCAPTCHA client.
+  client = ::Google::Cloud::RecaptchaEnterprise::V1::RecaptchaEnterpriseService::Client.new
```

Comment:
use:

```ruby
client = Google::Cloud::RecaptchaEnterprise.recaptcha_enterprise_service
```

In general, use these module methods defined in the wrapper client to create client objects, to avoid having to type these really long namespaces that include `V1` in them.